.flake8

```diff
@@ -35,8 +35,10 @@
 # - E203: white space around slice operators can be required, ignore : warn
 # - W503: see above, already ignored for line-breaks
 #
+# F821: generates an error on mypy comments with error codes like # ignore[misc]
```

Comment:
We don't want to ignore this globally since it means all undefined variables will be completely ignored.

It looks like this was a bug in older versions of pyflakes (https://github.com/PyCQA/pyflakes/issues/475), fixed in 2.2.0 (https://github.com/PyCQA/pyflakes/pull/455). Can you try a newer version of pyflakes/flake8 instead?

Reply:
Reinstated, and verified with a pipx installed copy on my machine. Thanks for the catch.